src/inc/clrnt.h

```diff
@@ -837,6 +837,13 @@ RtlVirtualUnwind_Unsafe(
 
 #ifdef _TARGET_X86_
 #ifndef FEATURE_PAL
+//
+// x86 ABI does not define RUNTIME_FUNCTION. Define our own to allow unification between x86 and other platforms.
```

Comment:
Sure, I tried not to affect windows x86 stuff via submitting small commits frequently (I have no mean to check Windows build issue).